app/src/main/java/cz/martykan/forecastie/activities/MainActivity.java

```diff
@@ -63,6 +63,8 @@
 import cz.martykan.forecastie.widgets.AbstractWidgetProvider;
 import cz.martykan.forecastie.widgets.DashClockWeatherExtension;
 
+import static cz.martykan.forecastie.utils.UnitConvertor.convertUvIndex;
```

Comment:
I wouldn't recommend using a static import. And the way you've written the code, it isn't really necessary. You can just normally import `UnitConverter`.